scripts/common/sonic-stp-build/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -xe
```

Comment:
Please change the file permission for the file, make it executable.
-rw-rw-r--
Change to
-rwxrwxr-x
See: https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-stp-build/2/console
Permission denied

Reply:
Done. taken care.